src/MLJDecisionTreeInterface.jl

```diff
@@ -70,40 +69,26 @@ function MMI.fit(m::DecisionTreeClassifier, verbosity::Int, X, y)
     fitresult = (tree, classes_seen, integers_seen, features)
 
     cache  = nothing
-    report = (classes_seen=classes_seen,
-              print_tree=TreePrinter(tree),
-              features=features,
-              )
+    report = (
+        classes_seen=classes_seen,
+        print_tree=TreePrinter(tree),
+        features=features,
+    )
     return fitresult, cache, report
 end
 
-function get_encoding(classes_seen)
-    a_cat_element = classes_seen[1]
-    return Dict(MMI.int(c) => c for c in MMI.classes(a_cat_element))
-end
+get_encoding(classes_seen) = Dict(MMI.int(c) => c for c in classes(classes_seen))
 
 MMI.fitted_params(::DecisionTreeClassifier, fitresult) =
     (tree=fitresult[1],
      encoding=get_encoding(fitresult[2]),
      features=fitresult[4])
 
-function smooth(scores, smoothing)
-    iszero(smoothing) && return scores
-    threshold = smoothing / size(scores, 2)
-    # clip low values
-    scores[scores .< threshold] .= threshold
-    # normalize
-    return scores ./ sum(scores, dims=2)
-end
-
```

Comment:
The function removed here is completely redundant. A leftover from when we had a pdf smoothing option.